content/opt/run

```diff
@@ -219,5 +212,5 @@ echo "==================================================================="
 touch ${initfile}
 fi
 
-echo "Starting Supervisor.  You can safely CTRL-C and the container will continue to run with or without the -d (daemon) option"
+echo "Starting Supervisor."
```

Comment:
I think it's because you're running the container with the -i interactive flag so you're actually sending the break to supervisor and not the host.